electric-operator.el

```diff
@@ -201,9 +201,15 @@ Whitespace before the operator is captured for possible use later.
        (-sort (lambda (p1 p2) (> (length (car p1)) (length (car p2)))) it)
        (car it)))
 
-(defun eval-action (action point)
+(defun eval-action (action point state)
   (cond
-   ((functionp action) (save-excursion (goto-char point) (funcall action)))
+   ((functionp action)
+    (save-excursion
+      (goto-char point)
+      ;; For backwards-compatibility only pass state to
```

Comment:
Not sure if there is a better way of doing this without breaking backwards-compatibility

Reply:
Stefan Monnier [thinks trying calling it both ways and catching the exception is the way to go](https://emacs.stackexchange.com/a/29247/237), but there doesn't seem to be a *good* way to handle this which is a shame. His problem with this version is that it won't handle some more advanced features like advised functions.

I think this is fine for now (unless you want to change it). I'll tweak it in the future if it causes problems.

Reply:
I found that we can do `((looking-back-locally "^\\s-*") "/")` instead of checking if the initial operator insertion point is at the indentation. It passes all the tests, and I can't think of anywhere that it won't work. So to avoid the extra complexity of having variable numbers of arguments I changed it to do that.

Can you think of any edge cases where that won't work?

Reply:
I had a reason yesterday which is why I replaced my initial version which was using `looking-back-locally` with `state`, however, I currently cannot remember. So most likely some unusual edge case.

Reply:
Ah ok, well if you run into the issue again we can try to fix it.

One weird thing I found while messing with the tests is that `c-electric-slash` etc actually modify the indentation when you type the second `/` (unless you've turned it off). Maybe that was causing some odd edge cases?

Reply:
I don't have c-electric-* turned on, so that shouldn't have been it. I'll take a note next time I come across it/something similar.